docs/pages/masonry.js

```diff
@@ -74,6 +75,21 @@ const getPins = () => {
   return Promise.resolve(pinList);
 };
 
+function gridComponent({ data }: { data: Pin, ... }) {
```

Comment:
@dangerismycat should we make this an actual React component? `function GridComponent`?

Reply:
Yes, good call. Given typical Masonry usage in Pinboard, I was thinking `comp` needed to be a simple function and would be called as a function internally, but that's wrong — it's rendered as a component. Updating now